generators/package/index.ts

```diff
@@ -214,13 +238,36 @@ module.exports = class extends HtmlGenerator {
       if (fs.existsSync(packageJsonPath)) {
         const packageJSONObject = this.fs.readJSON(packageJsonPath);
         if (packageJSONObject != null && packageJSONObject["description"] != null && packageJSONObject["description"] != "") {
-          packageInfo.push({type: "separator", line: `    ${packageJSONObject["description"]}`});
-          packageInfo.push({type: "separator", line: "        "});
+          packageInfo.push({ type: "separator", line: `    ${packageJSONObject["description"]}` });
+          packageInfo.push({ type: "separator", line: "        " });
         }
       }
       packagesWithDescription.push(...packageInfo);
     })
 
     return packagesWithDescription;
   }
+
+  /**
+   * Publish current package to the provided npm registry endpoint and channel.
+   */
+  private _publishPackageToNpmRegistry() {
+    let filePath = `${this.destinationPath(".dev-tasks.json")}`,
+      fileContent = this.fs.readJSON(this.destinationPath(filePath));
```

Comment:
destinationPath resolved twice?

Reply:
@joaoagoncalves can you take a look plz?

Reply:
already fixed this issue, I am testing this changes with the new bundles changes, I will then submit  the updated code.